cscore/CMakeLists.txt

```diff
@@ -138,27 +114,6 @@ if(WITH_JAVA)
     install_jar_exports(TARGETS cscore_jar FILE cscore_jar.cmake DESTINATION share/cscore)
     install(FILES ${OPENCV_JAR_FILE} DESTINATION "${java_lib_dest}")
 
-    if(MSVC)
-        install(FILES ${OPENCV_JNI_FILE} DESTINATION "${jni_lib_dest}")
```

Comment:
Is this no longer required for the JNI to load?

Reply:
It's not required. I've updated the CMake doc to mention that as long as the libraries are available on PATH, Java will be able to find the DLLs and load them.